libraries/hermes/src/Rest/Swagger.ts

```diff
@@ -1,17 +1,17 @@
 import { SwaggerParser } from './SwaggerParser';
-import { isNil } from 'lodash';
+import { isNil, cloneDeep } from 'lodash';
 import { ConduitRouteActions, Indexable, ConduitModel } from '@conduitplatform/grpc-sdk';
 import { SwaggerRouterMetadata } from '../types';
 import { ConduitRoute } from '../classes';
 import { importDbTypes } from '../utils/types';
 
 export class SwaggerGenerator {
-  private readonly _swaggerDoc: Indexable;
-  private readonly _routerMetadata: SwaggerRouterMetadata;
+  private _swaggerDoc: Indexable;
+  private _routerMetadata: SwaggerRouterMetadata;
   private readonly _stringifiedGlobalSecurityHeaders: string;
   private readonly _parser: SwaggerParser;
 
-  constructor(routerMetadata: SwaggerRouterMetadata) {
+  constructor(private readonly routerMetadata: SwaggerRouterMetadata) {
```

Comment:
Rename `routerMetadata` to `_routerMetadataOriginal`/`_declaredRouterMetadata`/??? to avoid unnecessary confusion over which object is actually used or why there's two of them.